ibm/service/iamidentity/resource_ibm_iam_trusted_profile_identity.go

```diff
@@ -0,0 +1,209 @@
+// Copyright IBM Corp. 2023 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package iamidentity
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/iamidentityv1"
+)
+
+func ResourceIBMIamTrustedProfileIdentity() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMIamTrustedProfileIdentityCreate,
+		ReadContext:   resourceIBMIamTrustedProfileIdentityRead,
+		DeleteContext: resourceIBMIamTrustedProfileIdentityDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"profile_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "ID of the trusted profile.",
+			},
+			"identity_type": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_iam_trusted_profile_identity", "identity_type"),
+				Description:  "Type of the identity.",
+			},
+			"identifier": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Identifier of the identity that can assume the trusted profiles. This can be a user identifier (IAM id), serviceid or crn. Internally it uses account id of the service id for the identifier 'serviceid' and for the identifier 'crn' it uses account id contained in the CRN.",
+			},
+			"type": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_iam_trusted_profile_identity", "type"),
+				Description:  "Type of the identity.",
+			},
+			"accounts": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Only valid for the type user. Accounts from which a user can assume the trusted profile.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"description": &schema.Schema{
```

Comment:
change in description is force_new (which destroy and recreate the resource)

Reply:
accommodated have been accomodated.

Reply:
What changes were accomodated we still see 
ForceNew:    true,?